### PR TITLE
Fix long preset names not looking correct

### LINKED ISSUE #6468: Long preset names look broken in the popup
### Description

If an addon has a preset with a long name, the button isn't displayed correctly in the popup.

### Steps to reproduce

There isn't a long enough preset name in the current version of SA (at least not in English), but #6467 adds one (see screenshot).

### Demonstration(s) and additional context

image

### Environment

Browser: Edge 117 dev
Scratch Addons version: 2f20ba0
Operating system: Windows

### PR DESCRIPTION
**Pull request by mybearworld**
Resolves #6468

(Don't mind the branch name, I originally solved this using a different strategy which didn't actually work)

### Changes

Change the way padding works on a .large-button to use actual padding.

### Reason for changes

So that buttons with more than one line look correct

### Tests

Tested in Edge, it would surprise me if it didn't work everywhere

the button looking correct

### CODE REVIEW

**Comment by WorldLanguages** [comment]
Make sure this doesn't change the style of the buttons in the normal settings view.

**Comment by mybearworld** [comment]
> Make sure this doesn't change the style of the buttons in the normal settings view.

It does not. Top is this PR, bottom is how it is currently.

image
image

**Review by DNin01 — COMMENTED** [review]
Much better, but there is still an issue visible in your screenshot - on the button with the wrapping text, the color palette and the next line of text are shifted slightly.

**Comment by mybearworld** [comment]
> Much better, but there is still an issue visible in your screenshot - on the button with the wrapping text, the color palette and the next line of text are shifted slightly.

Fixed.

File: webpages/styles/components/buttons.css [hunk 1]
```diff
@@ -2,10 +2,10 @@
   color: var(--content-text);
   background: var(--button-background);
   border: 1px solid var(--control-border);
-  padding: 0 12px;
+  display: flex;
```
Comment (mxmou) [inline webpages/styles/components/buttons.css:5]: This makes the preview and text incorrectly aligned unless `vertical-align` is set to `center`.

Comment (mybearworld) [reply webpages/styles/components/buttons.css:5]: True, I didn't notice.
```suggestion
  display: flex;
  align-items: center;
```

**Review by mxmou — APPROVED** [review]

**Pull request merged**